utility/map_str_enhancer.go

```diff
@@ -133,3 +137,12 @@ func MillisAsMicros(ms float64) common.MapStr {
 	m["us"] = int(ms * 1000)
 	return m
 }
+
+func Prune(m common.MapStr) common.MapStr {
```

Comment:
When using `utility.Add` this should actually not be necessary, as the `Add` method only adds key-value pairs if the value is not null or an empty object.

Reply:
This is used in a different context than `Add` normally is - here it's against `ManualDecoder.MapStr` result